### PR TITLE
PR from wdl/correct-attribute to wdl/dev to load attributes fields and values into game-state structures

### PR DESCRIPTION
**Pull request by bansals10**
This is a pull request to load attributes into the structures provided by the game-state team. In order to complete this, I created a while loop to iterate over the attributes within the attribute list from the document. I also added a function to the parse module to retrieve the attributes associated with a given item. More thorough integration tests will be written once this PR has been approved.

### CODE REVIEW

File: src/wdl/tests/test_item.c [hunk 1]
```diff
@@ -3,6 +3,24 @@
 #include <stdbool.h>
 #include "load_item.h"
 
+/* see item.h */
+char *get_sdesc_item(item_t *item)
```
Comment (mewthew) [inline src/wdl/tests/test_item.c]: Can you remove this? It should have already been added, you might just need to re-make the game-state library

File: src/wdl/tests/test_item.c [hunk 2]
```diff
@@ -3,6 +3,24 @@
...
+{
+  if (item == NULL) {
+    return NULL;
+  }
+  return item->short_desc;
+}
+
+/* see item.h */
+char *get_ldesc_item(item_t *item)
```
Comment (mewthew) [inline src/wdl/tests/test_item.c]: Same as above comment.

File: src/wdl/tests/test_item.c (hunk 1 above)
Comment (bansals10) [reply src/wdl/tests/test_item.c]: Done.

File: src/wdl/tests/test_item.c (hunk 2 above)
Comment (bansals10) [reply src/wdl/tests/test_item.c]: Done.

**Comment by bansals10** [comment]
I have updated the code by removing extraneous game-state item functions. After re-making the game-state library, the code compiles properly and the tests run as expected. I will write more thorough integration tests in a future issue for this code.

**Pull request merged**